shelley-ma/shelley-ma-test/src/Test/Cardano/Ledger/Mary.hs

```diff
@@ -85,21 +250,25 @@ genTxBody ::
   Coin ->
   StrictMaybe (Update era) ->
   StrictMaybe (AuxiliaryDataHash (Crypto era)) ->
-  Gen (TxBody era)
+  Gen (TxBody era, [Timelock (Crypto era)])
 genTxBody slot ins outs cert wdrl fee upd meta = do
   validityInterval <- genValidityInterval slot
-  let mint = error "TODO @uroboros mint some Mary era tokens"
+  mint <- genMint
```

Comment:
we can wrap `genMint` with a `genMintWithScripts` that produces the policy scripts present in it's generated token bundles